src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,74 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onChangePage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onChangePage,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+  const pages = new Array<number>(pagesAmount).fill(0).map((_, i) => i + 1);
```

Comment:
would be good to memoize this array:
```suggestion
  const pages = useMemo(() => new Array<number>(pagesAmount).fill(0).map((_, i) => i + 1), [pagesAmount]);
```